src/hoverboard-app.html

```diff
@@ -71,11 +71,14 @@
       }
 
       .drawer-list a {
-        padding: 8px 24px;
         display: block;
         color: var(--primary-text-color);
       }
 
+      app-drawer a {
```

Comment:
should it apply to all <a> elements, or only .buy-ticket?

Reply:
It's applied to .drawer-list a, but the same padding should be used to .buy-ticket